src/containers/AllSubjectsPage/utils.ts

```diff
@@ -0,0 +1,68 @@
+import groupBy from 'lodash/groupBy';
+import { GQLMySubjectsSubjectFragmentFragment } from '../../graphqlTypes';
+
+export const letters = [
```

Comment:
Skader sikkert ikke å kalle denne `subjectLetters` istedenfor, for å gi den litt mer kontekst.